CleanMotor:
  Alive: 1
  pub_rate: 25.0
  PowerLimit: 0.5
  GroupNum: 1
  DeviceNum: 1
  Mode: 3  # 1: open loop; 3: velocity; 4: position;5:vel-pos;


  Diameter: 40.0 #mm, brush head diameter
  BasicEncoderLines: 11
  ReductionRatio: 90


  P: 29.5
  I: 0
  D: 0

# Can_File: "/home/sgl/catkin_new/src/robomodule/config/can_config.yaml"
RCRMotor:
  pub_rate: 50.0
  Alive: 1
  PowerLimit: 1
  GroupNum: 0
  DeviceNum: 1
  Mode: 5  # 1: open loop; 3: velocity; 4: position;5:vel-pos;


  Diameter: 508.0 #40*12.7mm, brush head diameter
  BasicEncoderLines: 1500
  ReductionRatio: 40


  P: 49.6
  I: 0
  D: 0